TombEngine/Game/Lara/lara_struct.h

```diff
@@ -822,14 +824,21 @@ enum LaraAnim
 	LA_LEDGE_JUMP_UP_END = 566,
 	LA_LEDGE_JUMP_BACK_START = 567,
 	LA_LEDGE_JUMP_BACK_END = 568,
-
+	
 	// 569-598 reserved for ladder object. -- Sezz 2023.04.16
 
+	LA_UNDERWATER_WALL_KICK = 599,			// Lara kick wall underwater.
+	LA_WALL_PUSH = 600,						// Lara Push Wall.
+	LA_WALL_LEVER_SWITCH = 601,				// Use underwater level above ground
```

Comment:
Can we add `LA_UNDERWATER...` to all new underwater animations? This name is misleading because it doesn't state that it's used underwater.